### PR TITLE
update dataloader

### PR DESCRIPTION
**Pull request by XitongZhang1994**
fix the data loader for cora_ml and citeseer
add train/validation/test masks

### CODE REVIEW

**Comment by SherylHYX** [comment]
# [Codecov](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He) Report
> Merging [#3](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He) (666a7d4) into [main](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/commit/c8ba9d84cb7cf8903b213f1a67809f02deb3fd3d?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He) (c8ba9d8) will **decrease** coverage by `0.47%`.
> The diff coverage is `88.78%`.

[Impacted file tree graph](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He)

```diff
@@            Coverage Diff             @@
##             main       #3      +/-   ##
==========================================
- Coverage   98.93%   98.45%   -0.48%     
==========================================
  Files          45       46       +1     
  Lines        1779     1879     +100     
==========================================
+ Hits         1760     1850      +90     
- Misses         19       29      +10     
```


| [Impacted Files](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He) | Coverage Δ | |
|---|---|---|
| [...gned\_directed/data/signed/snap\_signed\_real\_data.py](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He#diff-dG9yY2hfZ2VvbWV0cmljX3NpZ25lZF9kaXJlY3RlZC9kYXRhL3NpZ25lZC9zbmFwX3NpZ25lZF9yZWFsX2RhdGEucHk=) | `100.00% <ø> (ø)` | |
| [...metric\_signed\_directed/data/directed/node\_split.py](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He#diff-dG9yY2hfZ2VvbWV0cmljX3NpZ25lZF9kaXJlY3RlZC9kYXRhL2RpcmVjdGVkL25vZGVfc3BsaXQucHk=) | `86.95% <86.95%> (ø)` | |
| [...\_directed/data/directed/load\_directed\_real\_data.py](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He#diff-dG9yY2hfZ2VvbWV0cmljX3NpZ25lZF9kaXJlY3RlZC9kYXRhL2RpcmVjdGVkL2xvYWRfZGlyZWN0ZWRfcmVhbF9kYXRhLnB5) | `97.74% <100.00%> (+0.14%)` | :arrow_up: |
| [...tric\_signed\_directed/data/signed/polarized\_SSBM.py](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He#diff-dG9yY2hfZ2VvbWV0cmljX3NpZ25lZF9kaXJlY3RlZC9kYXRhL3NpZ25lZC9wb2xhcml6ZWRfU1NCTS5weQ==) | `100.00% <0.00%> (+1.44%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3?src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3?src=pr&el=footer&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He). Last update [c8ba9d8...666a7d4](https://codecov.io/gh/SherylHYX/pytorch_geometric_signed_directed/pull/3?src=pr&el=lastupdated&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Yixuan+He).
